lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -85,18 +81,39 @@ final class ViewFocusBinding {
         direction: _viewFocusDirection,
       );
     }
+    _markViewAsFocusable(_lastViewId, reachableByKeyboard: true);
+    _markViewAsFocusable(viewId, reachableByKeyboard: false);
     _lastViewId = viewId;
-    _notify(event);
+    _onViewFocusChangeController.add(event);
   }
 
-  static int? _viewId(DomElement? element) {
-    final DomElement? viewElement = element?.closest(
-      DomManager.flutterViewTagName,
-    );
-    final String? viewIdAttribute = viewElement?.getAttribute(
-      GlobalHtmlAttributes.flutterViewIdAttributeName,
-    );
-    return viewIdAttribute == null ? null : int.tryParse(viewIdAttribute);
+  int? _viewId(DomElement? element) {
+    final DomElement? rootElement = element?.closest(DomManager.flutterViewTagName);
```

Comment:
Agh, I thought we were setting the focusin/focusout listeners in the rootElement 🤦

Reply:
Yeah, easy to miss; but no, all those events are attached to the body.